src/chapters/sql-part1/queries.rst

```diff
@@ -0,0 +1,256 @@
+      GuestID int,
+      LastName varchar(255),
+      FirstName varchar(255),
+      Attending BOOL,
+      Diet varchar(255)  
+   );
+
+This query creates a new table in our database called ``LiWedding``. This table contains five columns: ``GuestID`` of type ``int``, ``LastName`` of type ``varchar(255)``, ``FirstName`` of type ``varchar(255)``, ``Attending`` of type ``BOOL``, ``Diet`` of type ``varchar(255)``.
+When we use ``CREATE TABLE`` to create a table from scratch, we will end up with an empty table. We will learn how to add to a table a little further down the page.
+
+Creating a Table from Another Table
+"""""""""""""""""""""""""""""""""""
+
+We can also create a table from another table.
+When doing so, the new table will also include any existing data from the original table.
+
+Mary is also planning a vow renewal for Mr. and Mrs. Johnson.
+Since the Johnsons haven't made any new friends in the past 5 years, they want the guest list to be the exact same.
+A good place to start would be to create a table called ``JohnsonVowRenewal`` from the table called ``JohnsonWedding``.
+Due to some wedding drama, Mr. and Mrs. Johnson will also be avoiding inviting anyone who didn't show up for the first wedding.
+
+.. sourcecode:: mysql
+   :linenos:
+
+   CREATE TABLE JohnsonVowRenewal
+   AS GuestID, LastName, FirstName, Attending, Diet
+   FROM JohnsonWedding
+   WHERE (Attending == 1);
+
+This query creates the ``JohnsonVowRenewal`` table with the same columns as ``JohnsonWedding``.
+We use ``AS`` to specify the columns that will be carried over from the ``JohnsonWedding`` table to the ``JohnsonVowRenewal``.
+The ``WHERE`` condition in the query specifies that we only want the rows from ``JohnsonWedding`` in ``JohnsonVowRenewal`` *where* the guest was in attendance (or ``Attending == 1``).
+If we did *not* add a ``WHERE`` condition, then we would have copied over all of the data from ``JohnsonWedding``.
+
+Adding a Row
+^^^^^^^^^^^^
+
+Whether we have created a table from scratch or are working with an existing table, we may need to add a row of data.
+To do so, we need to use an ``INSERT INTO`` statement.
+
+In the case of the Johnsons, we may need to add a row for their niece, who is now old enough to attend the vow renewal.
+Their niece is Eliza Johnson and she is a vegan. Her mother has also confirmed that she will be in attendance.
+
+.. sourcecode:: mysql
+
+   INSERT INTO JohnsonVowRenewal
+   VALUES (185, "Johnson", "Eliza", 1, "Vegan");
+
+This query adds a row for Eliza to the ``JohnsonVowRenewal`` table in our database.
+
+If we wanted to add a row, but only add values to specific columns in the table, we can do so! We simply need to add the names of the columns that we will be adding data to.
+
+In the case of inviting people the Johnson's vow renewal, we may want to invite Eliza's sister, Felicity, as well. However, we do not have confirmed Felicity's dietary preferences or whether or not she is coming.
```

Comment:
Phrasing update.  "However, we have not confirmed..."

Reply:
Done! Thanks!